app/code/community/Oyst/OneClick/Helper/Data.php

```diff
@@ -173,4 +173,30 @@ public function isPaymentMethodOyst(Mage_Sales_Model_Order $order)
 
         return false !== strpos($order->getPayment()->getMethod(), $freepayPaymentMethod->getCode());
     }
+
+    /**
+     * Get metadata string.
+     *
+     * @return null|string
+     */
+    public function getTrackingMeta()
+    {
+        if (!Mage::getStoreConfig('oyst_oneclick/oneclick/tracking')) {
+            return null;
+        }
+
+        return '<script src="https://trk.10ru.pt"></script>' . PHP_EOL
+            . '<meta name="oyst_tracker_info_freepay_activated" content="'
+            . (int)Mage::getStoreConfigFlag('payment/oyst_freepay/active') . '">' . PHP_EOL
+            . '<meta name="oyst_tracker_info_freepay_apikey" content="'
+            . (int)Mage::getStoreConfigFlag('payment/oyst_abstract/api_login') . '">' . PHP_EOL
+            . '<meta name="oyst_tracker_environnement_freepay" content="'
+            . Mage::getStoreConfig('payment/oyst_abstract/mode') . '">' . PHP_EOL
+            . '<meta name="oyst_tracker_info_oneclick_activated" content="'
+            . (int)Mage::getStoreConfigFlag('oyst/oneclick/enable') . '">' . PHP_EOL
+            . '<meta name="oyst_tracker_info_oneclick_apikey" content="'
+            . (int)Mage::getStoreConfigFlag('oyst/oneclick/api_login') . '">' . PHP_EOL
```

Comment:
@rus-daniel I just delete in database oyst/oneclick/api_login and I get 1 value, could you check?